### PR TITLE
Project 1-3: Added a RegisterServlet class

### PR DESCRIPTION
**Pull request by anujinm**

### CODE REVIEW

**Comment by kuanyiwu** [comment]
Looks good!

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
```
Comment (aditija) [inline src/main/java/codeu/controller/RegisterServlet.java]: It looks like the class doesn't do anything yet right? Want to add a comment that calls this out?

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+   response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
+ }
+}
```
Comment (aditija) [inline src/main/java/codeu/controller/RegisterServlet.java]: This looks like they want you to add a newline?

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+   response.getWriter().println("<h1>RegisterServlet GET request.</h1>");
```
Comment (aditija) [inline src/main/java/codeu/controller/RegisterServlet.java]: It looks like the class doesn't do anything yet right? Want to add a comment that calls this out?

Comment (anujinm) [reply src/main/java/codeu/controller/RegisterServlet.java]: Yes, it just prints RegisterServlet GET request.  I thought that's what they wanted for now.
Here is the ss of what it does for now.
image

Comment (aditija) [reply src/main/java/codeu/controller/RegisterServlet.java]: Correct I was just saying you can add a comment under

Servlet class responsible for user registration.

that says something like

Note: User registration is not yet implemented. For now, the servlet just prints to the browser.

File: src/main/java/codeu/controller/RegisterServlet.java (hunk 2 above)
Comment (anujinm) [reply src/main/java/codeu/controller/RegisterServlet.java]: Oh, gotcha. But I think it's  not necessary anymore, because I modified it in the next step.

**Comment by anujinm** [comment]
I'll just this pull request and make a new one with new changes.

**Pull request closed without merging**